crypto/curve_test.go

```diff
@@ -0,0 +1,31 @@
+package crypto
+
+import (
+	"encoding/hex"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestBLS12381Compatv112(t *testing.T) {
```

Comment:
What's this ?

Reply:
A test coming from the `key` package, moved here instead.